npm/private/npm_import.bzl

```diff
@@ -503,7 +503,12 @@ def _npm_import_rule_impl(rctx):
 
     # apply patches to the extracted package before reading the package.json incase
     # the patch targets the package.json itself
-    patch(rctx, patch_args = rctx.attr.patch_args, patch_directory = _EXTRACT_TO_DIRNAME)
+    patch(
+        rctx,
+        patch_tool = rctx.path(rctx.attr.patch_tool) if rctx.attr.patch_tool else "patch",
```

Comment:
we should consider ALWAYS using a hermetic patch implementation. I recall we looked into the one Bazel itself uses, but it wasn't exposed for repository rules to use? Maybe @gregmagolan remembers. Can be in a later PR though.